inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -425,6 +393,92 @@ protected function get_absolute_url( $src ) {
 	}
 
 	/**
+	 * Returns the attachments for a gallery.
+	 *
+	 * @param int   $id      The post id.
+	 * @param array $gallery The gallery config.
+	 *
+	 * @return array The selected attachments
+	 */
+	protected function get_gallery_attachments( $id, $gallery ) {
+
+		// When there are posts to include.
+		if ( ! empty( $gallery['include'] ) ) {
+			$_attachments = $this->get_attachments(
+				array(
+					'include'     => $gallery['include'],
```

Comment:
`include` is not a valid argument for WP_Query. Use `post__in` instead, the value for this will need to be parsed with `wp_parse_id_list`.